e2e/tests/gatekeeperAdministersCaseAfterSubmission_test.js

```diff
@@ -5,7 +5,7 @@ let caseId;
 
 Feature('Gatekeeper Case administration after submission');
```

Comment:
Probably that feature and file name is not correct anymore, would be good to have this changed.